config/dev.exs

```diff
@@ -3,9 +3,9 @@ use Mix.Config
 # Configure your database
 config :ag, AG.Repo,
   username: "postgres",
-  password: "postgres",
+  password: "",
   database: "ag_dev",
-  hostname: "localhost",
+  hostname: "db",
```

Comment:
Do we need to change this?

Reply:
docker-compose created its own network and it must be the same as this line https://github.com/jakyns/admiring-gamification/pull/8/files#diff-4e5e90c6228fd48698d074241c2ba760R7 that's why